src/microbe_stage/MicrobeHUD.cs

```diff
@@ -720,6 +737,8 @@ private void UpdateReproductionProgress()
 
         ammoniaReproductionBar.Value = fractionOfAmmonia * ammoniaReproductionBar.MaxValue;
         phosphateReproductionBar.Value = fractionOfPhosphates * phosphateReproductionBar.MaxValue;
+        OnAmmoniaBarValueChanged();
```

Comment:
I think you should just pass the fractionOfAmmonia as a parameter here.
So that in that method you can then just have:
```
if(fractionOfAmmonia < 1.0f)
    return;
```

And maybe rename it, CheckAmmoniaProgressHighlight.